CHANGELOG.md

```diff
@@ -4,6 +4,27 @@
 <!--- Distributed under the Boost Software License, Version 1.0. (See accompanying -->
 <!--- file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt) -->
 
+## 0.13.0 (2023-03-08)
+
+### New features
+
+- Add `static_assert` to have better error messages with senders. ([#586](https://github.com/pika-org/pika/pull/586))
+
+### Breaking changes
+
+- Rename `done_*` with `stopped_*` to conform with stdexec implementation. ([#583](https://github.com/pika-org/pika/pull/583))
+- Remove `PIKA_WITH_BACKGROUND_THREAD_COUNTERS`. ([#588](https://github.com/pika-org/pika/pull/588))
+- Update stdexec commit used. ([#597](https://github.com/pika-org/pika/pull/597))
+- Update moodycamel concurrent queue to the latest version. ([#608](https://github.com/pika-org/pika/pull/608))
+- Cleaned up modules and moved minor functionality into `detail` namespaces. ([#607](https://github.com/pika-org/pika/pull/607), [#607](https://github.com/pika-org/pika/pull/607))
+
+### Bugfixes
+
+- Initialize HIP early to avoid concurrent initialization. ([#591](https://github.com/pika-org/pika/pull/591))
+- Minor fix in `cuda_sender_test`. ([#592](https://github.com/pika-org/pika/pull/592))
+- fix warning about no return in `unique_any_sender` connect overload. ([#594](https://github.com/pika-org/pika/pull/594), [#595](https://github.com/pika-org/pika/pull/595), [#596](https://github.com/pika-org/pika/pull/596), [#599](https://github.com/pika-org/pika/pull/599), [#605](https://github.com/pika-org/pika/pull/605))
```

Comment:
Since this warning was introduced after the latest release I don't think we need to include it (it was introduced by #586).
```suggestion
```